src/content/docs/en/reference/content-loader-reference.mdx

```diff
@@ -290,7 +287,7 @@ export function articleLoader(config: { apiKey: string }): LiveLoader<Article> {
 }
 ```
 
-<ReadMore>[See the `Content Collection` guide](/en/guides/content-collections/#creating-a-live-loader) for more information about creating a live loader and example usage.</ReadMore>
+<ReadMore>[See the “Content Collections” guide](/en/guides/content-collections/#creating-a-live-loader) for more information about creating a live loader and example usage.</ReadMore>
```

Comment:
```suggestion
<ReadMore>See [how to create a live loader](/en/guides/content-collections/#creating-a-live-loader) with example usage.</ReadMore>
```
Note: we don't "air quote" the names of our pages at all! But it's even better if we have more descriptive link text about what is at the link.